packages/datetime/examples/dateRangeInputExample.tsx

```diff
@@ -42,6 +47,12 @@ export class DateRangeInputExample extends BaseExample<IDateRangeInputExampleSta
     protected renderOptions() {
         return [
             [
+                <FormatSelect
```

Comment:
Made this column narrower to make room for longer labels in the toggle column. Felt better on the left side.